pool/app/contact_counter/contact_counter.ml

```diff
@@ -0,0 +1,85 @@
+let update_on_invitation_sent m =
+  Contact.
+    { m with
+      num_invitations = m.num_invitations |> NumberOfInvitations.increment
+    }
+;;
+
+let update_on_session_signup m sessions =
```

Comment:
Same as `Entity.increment_num_assignments`

Reply:
It takes an argument, by hot many to increment